src/main/webapp/app/exam/manage/students/upload-images/students-upload-images-dialog.component.ts

```diff
@@ -61,7 +60,7 @@ export class StudentsUploadImagesDialogComponent implements OnDestroy {
 
     private resetDialog() {
         this.isParsing = false;
-        this.notFoundUsers = undefined;
+        this.notFoundUsers = {} as NotFoundExamUserType;
```

Comment:
This does not seem correct to me. Accrding to the types, `numberOfUsersNotFound` cannot be undefined, so `this.notFoundUsers .numberOfUsersNotFound` should also not be undefined. But with this assignement this is possible.

Reply:
Yes you are right. I reverted the change back anyways so this field can be undefined.